src/source/raster_tile_source.js

```diff
@@ -112,41 +113,32 @@ class RasterTileSource extends Evented implements Source {
     loadTile(tile: Tile, callback: Callback<void>) {
         const use2x = browser.devicePixelRatio >= 2;
         const url = this.map._requestManager.normalizeTileURL(tile.tileID.canonical.url(this.tiles, this.scheme), use2x, this.tileSize);
-        tile.request = getImage(this.map._requestManager.transformRequest(url, ResourceType.Tile), (err, img, cacheControl, expires) => {
+        tile.request = getImage(this.map._requestManager.transformRequest(url, ResourceType.Tile), (error, data, cacheControl, expires) => {
             delete tile.request;
 
-            if (tile.aborted) {
-                tile.state = 'unloaded';
-                callback(null);
-            } else if (err) {
-                tile.state = 'errored';
```

Comment:
It looks like we're not setting 'unloaded' or 'errored' states anymore. Is this intentional?

Reply:
When I search for `errored` or `tile.state`, a bunch of uses popup. I didn't trace through the effects of removing these, but we should be really confident that this isn't going to break anything if we remove it.

Reply:
It looks like the `errored` state is used in a few places to help determine if the tile is loaded. See https://github.com/mapbox/mapbox-gl-js/blob/main/src/ui/map.js#L1879 for example

Reply:
Yep, I missed that, my bad